CMakeLists.txt

```diff
@@ -43,6 +43,15 @@ if(FLOW_BENCHMARKING)
   file(MAKE_DIRECTORY "${FLOW_BENCHMARK_DIR}")
 endif()
 
+if(HOLOHUB_BUILD_PYTHON)
+  # Define and create a directory to store python operators under the holohub python module
+  if(NOT CMAKE_INSTALL_LIBDIR)
```

Comment:
I guess it could be changed by the user, not sure if that was a request for the CLI.

Reply:
Seems like this block will only have effect if the user intentionally changes the value to an empty string, right?

If we have a requirement to enforce `lib` then we should always forcibly update the value. But, IIRC the CLI is soon adding an option to explicitly set library directoreis to use, so we probably don't need to force `lib`.

Reply:
No if the user provide another CMAKE_INSTALL_LIBDIR such as lib64 (which is the case for some distro) then it will use that variable.

Reply:
Yes, that's my point:
- If there is no requirement to enforce a certain library folder naming scheme, we should remove lines 48-50
- If there is a requirement to enforce a certain library folder naming scheme, then we should replace lines 48-50 with that setting

Reply:
I'm not sure I understand. If we remove line 48-50, how should we allow a user to set this variable?

Reply:
Typical workflow: `cmake -S <src> -B <build> -DCMAKE_INSTALL_LIBDIR:STRING="mylibdir"

The only way the block in line 48-50 might run is if the user explicitly sets `-DCMAKE_INSTALL_LIBDIR:STRING=""`, which would be a strange edge case but signals intentionality. Otherwise the default will be `lib`.

Overall this is more discussion than I meant to initiate, let's just stick with the existing code that represents the status quo, it's not breaking anything.